core/src/smartcontracts/wasm.rs

```diff
@@ -239,7 +239,7 @@ impl<'a, W: WorldTrait> Runtime<'a, W> {
         // Store length as byte array in front of encoding
         for (i, byte) in WasmUsize::try_from(r.len())
             .map_err(|e| Trap::new(e.to_string()))?
-            .to_be_bytes()
+            .to_le_bytes()
```

Comment:
Little endian for consistency with SCALE?

Reply:
yeah